creds/src/lib.rs

```diff
@@ -274,6 +287,100 @@ pub fn create_client_state(paths : &CachePaths, prover_inputs: &GenericInputsJSO
     Ok(client_state)
 }
 
+
```

Comment:
Could we put all this wasm-specific code in a separate file? wasm_lib.rs or something?

Reply:
Put the wasm code into its own file.
We still need a few wasm references outside of the new file.